docs/en/latest/terminology/secret.md

```diff
@@ -168,6 +168,12 @@ secrets:
     uri: 127.0.0.1:8200
 ```
 
+:::tip
+
+It now supports the use of the `namespace` field to set the multi-tenant namespace concepts supported by HashiCorp Vault Enterprise and HCP Vault.
```

Comment:
How about adding a link to the HC doc for the namespace, e.g.

```suggestion
It now supports the use of the [`namespace` field](https://developer.hashicorp.com/vault/docs/enterprise/namespaces#vault-api-and-namespaces) to set the multi-tenant namespace concepts supported by HashiCorp Vault Enterprise and HCP Vault.
```

Reply:
Yeah, that's better, updating.